hackathon_site/event/jinja2/event/landing.html

```diff
@@ -12,299 +14,365 @@
+                </div>
             </div>
+        </div>
+    </div>
+    <br /><br /><br />
 
-            <div class="col s12 m6">
-                <ul class="collapsible">
-                    <li>
-                        <div class="collapsible-header">
-                            Who can apply? What if I have no experience?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>
-                                Applicants that are currently in either highschool or university are welcomed to apply! We are a beginner friendly hackathon, so no hackathon experience is needed to
-                                participate! We are all here to learn :)
-                            </span>
-                        </div>
-                    </li>
-                    <li>
-                        <div class="collapsible-header">
-                            What equipment and support would be provided?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>
-                                Unfortunately, due to the current situation of COVID19. We won’t be able to provide any hardware. However, there would be lots of mentors ready to help you! During the
-                                online hackathon, there would be a breakout room for all the mentors where you would be able to access at all times and get immediate help.
-                            </span>
-                        </div>
-                    </li>
-                    <li>
-                        <div class="collapsible-header">
-                            How do online hackathons work?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>
-                                The entire hackathon would be conducted online. You would be able to stay safe and comfortably at home while doing this hackathon. You would be able to communicate to
-                                your team via online meetings and you would be submitting your final products online.
-                            </span>
-                        </div>
-                    </li>
-                </ul>
-            </div>
+    <div class="container section colorScroll" data-background-color="#EFCF5D">
+        <div class="s12 m8 flexColCenter">
+            <h3 class="center">Check out last year!</h3>
+            <iframe class="video center" src="https://www.youtube.com/embed/KPeTfGlh9DE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
         </div>
+        <br /><br />
+    </div>
+    <br /><br /><br />
+
+    <div class="carousel">
+        <!-- IMAGES MUST ALL BE THE SAME SIZE -->
+        <a class="carousel-item" ><img src="https://i.imgur.com/u9JEte7.jpg" alt="2020-image-1"/></a>
```

Comment:
These make the page load really slowly, can we shrink them all down to 500px height or smaller?

Reply:
done